frontend/src/pages/modelServing/useInferenceServices.ts

```diff
@@ -16,7 +16,9 @@ const useInferenceServices = (namespace?: string): FetchState<InferenceServiceKi
     return getInferenceServiceContext(namespace, LABEL_SELECTOR_DASHBOARD_RESOURCE);
   }, [namespace, modelServingEnabled]);
 
-  return useFetchState<InferenceServiceKind[]>(getServingInferences, []);
+  return useFetchState<InferenceServiceKind[]>(getServingInferences, [], {
+    initialPromisePurity: true,
+  });
```

Comment:
This is the way to make a `loaded` state refresh, so we can render the spinner when switching projects. Otherwise, the `loaded` won't change and we could never render a spinner.